src/main/resources/application-dev.yaml

```
@@ -0,0 +1,36 @@
+  access-token:
+    secret: ${JWT_ACCESS_TOKEN_SECRET:ef20dc1ba114dd1c3ee9d035785a551c8e99da05f642f67f2a0ea8643fe6996c}
+    expiration: ${JWT_ACCESS_TOKEN_EXPIRATION:86400}
+  refresh-token:
+    secret: ${JWT_REFRESH_TOKEN_SECRET:b093c643373ccf233b834c70a6eab2c1d48140785f0bb267eb8e3fca24fa2de9}
+    expiration: ${JWT_REFRESH_TOKEN_EXPIRATION:604800}
+
+coolsms:
+  key: ${COOLSMS_API_KEY:NCSUKVBHPYJWMYZE}
+  secret: ${COOLSMS_API_SECRET:87T9UCIIHRWOVGHN48RGCWCSFL27UOEI}
+  sender: ${COOLSMS_SENDER:01055736982}
+  ttl: ${COOLSMS_TTL:300}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)